### ISSUE TITLE
Bug report: submission: listening on tls://0.0.0.0:465 not open 

### ISSUE THREAD

**Issue by xiliuya** [foxcpp/maddy#671]
# Describe the bug
Test with thunderbird , smtp 465/tls not work,but 587/startls is fine.
Other functions are working properly.Is this a normal phenomenon?
When I run `nmap -Pn x.x.x.x`
```
22/tcp    open   ssh
25/tcp    open   smtp
80/tcp    open   http
143/tcp   open   imap
443/tcp   open   https
465/tcp   closed smtps
587/tcp   open   submission
993/tcp   open   imaps
```

# Steps to reproduce
```sh
podman run --name maddy \
                 -e MADDY_HOSTNAME=mx.domain \
                 -e MADDY_DOMAIN=domain \
                 -v /etc/maddy/:/data \
                 -p 25:25 \
                 -p 143:143 \
                 -p 587:587 \
                 -p 993:993 \
                 foxcpp/maddy:latest
```

* maddy version: **0.7.1 linux/amd64 go1.19.13**

**Comment by xiliuya** [comment]
I think the commands in the document need to be changed to:

```
TL;DR

docker volume create maddydata
docker run \
  --name maddy \
  -e MADDY_HOSTNAME=mx.maddy.test \
  -e MADDY_DOMAIN=maddy.test \
  -v maddydata:/data \
  -p 25:25 \
  -p 143:143 \
  -p 465:465 \
  -p 587:587 \
  -p 993:993 \
  foxcpp/maddy:0.6
```
